quilt/tools/package.py

```diff
@@ -217,8 +217,11 @@ def get_obj(self, node):
         if isinstance(node, TableNode):
             return self._dataframe(node.hashes, node.format)
         elif isinstance(node, GroupNode):
-            hash_list = [h for c in node.preorder_tablenodes() for h in c.hashes]
-            return self._dataframe(hash_list, PackageFormat.PARQUET)
+            try:
+                hash_list = [h for c in node.preorder_tablenodes() for h in c.hashes]
+                return self._dataframe(hash_list, PackageFormat.PARQUET)
+            except ValueError:
```

Comment:
Which line is throwing the `ValueError`? If possible, keep just that one line inside of `try`, to minimize false positives.